model/src/test/kotlin/licenses/LicenseInfoResolverTest.kt

```diff
@@ -448,3 +558,18 @@ fun containLocationForLicense(
             "ResolvedLicenseInfo should not contain location ${expectedLocation.show().value} for $license"
         )
     }
+
+fun ResolvedLicenseInfo.pathExcludesForLicense(license: String, provenance: Provenance, location: TextLocation) =
+    find { it.license == SpdxSingleLicenseExpression.parse(license) }
+        ?.locations
+        ?.find { it.provenance == provenance && it.location == location }
+        ?.matchingPathExcludes
+        ?.toSet().orEmpty()
+
+fun ResolvedLicenseInfo.pathExcludesForCopyright(copyright: String, provenance: Provenance, location: TextLocation) =
```

Comment:
I think not, because the path exclude does not really match the copyright finding, which would be hard to understand. The license location that references the copyright location will already be marked as excluded, that should be sufficient.